tests/e2e/get-integ-test-resources.js

```diff
@@ -38,7 +38,7 @@ exports.getIntegTestResources = async () => {
 
   const sts = new STSClient({ logger });
   const { Account: AccountId } = await sts.send(new GetCallerIdentityCommand({}));
-  const s3Control = new S3ControlClient({ logger });
+  const s3Control = new S3ControlClient({ logger, region: "us-west-2" });
```

Comment:
This allows the test stack to run in us-east-1. S3 control only wants us-west-2 for MRAP.